src/interpreter.ts

```diff
@@ -845,13 +845,13 @@ export class Interpreter<
 
     if (resolvedOptions.sync) {
       childService.onTransition(state => {
-        this.send(actionTypes.update, { state, id: childService.id });
+        this.send(actionTypes.update as any, { state, id: childService.id });
```

Comment:
seems like OmniEvent should include platform errors, done events and special update action